sklearn/linear_model/_ridge.py

```diff
@@ -245,11 +245,11 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
 
     Parameters
     ----------
-    X : {array-like, sparse matrix, LinearOperator} of shape \
+    X : {ndarray, sparse matrix, LinearOperator} of shape \
```

Comment:
For input it's fine to keep array-like, as anything that's not an ndarray (or one of the above options) will likely be converted to ndarray.